pkg/metallb/metallb.go

```diff
@@ -392,3 +393,101 @@ func (builder *Builder) convertToStructured(unsObject *unstructured.Unstructured
 
 	return metalLb, err
 }
+
+// WithMetallbControllerConfig defines the controller config with a helm chart table.
+func (builder *Builder) WithMetallbControllerConfig(priorityClassName, runtimeClassName, deploymentName,
```

Comment:
Please this func to other global With functions.

Reply:
moved